packages/office-ui-fabric-react/src/components/Toggle/Toggle.test.tsx

```diff
@@ -23,15 +23,15 @@ describe('Toggle', () => {
   });
 
   it('renders aria-label', () => {
-    const component = mount(<Toggle label="Label" offAriaLabel="offLabel" />);
+    const component = mount(<Toggle label="Label" ariaLabel="AriaLabel" />);
 
```

Comment:
We should move all tests using the deprecated props into a Toggle.deprecated.test.tsx and create new tests that use the newly supported props.

Reply:
Fixed? I think